### PR TITLE
Book Store : Add styling

### PR DESCRIPTION
**Pull request by forLoop94**
1. I added all missing JSX elements to the project (without adding extra functionality)
2. The final design matches Zeplin's design

### CODE REVIEW

File: src/styles/BookList.module.css [hunk 1]
```diff
@@ -5,34 +5,108 @@
...
+  height: 4rem;
+  margin-right: 1rem;
+  border-radius: 50%;
+}
+
+small {
+  color: rgb(155, 152, 152);
+}
+
+.blue {
```
Comment (Ghiftee) [inline src/styles/BookList.module.css:95]: - [ ] Kindly ensure that the following highlighted elements in the image have the same color as in the [zeplin design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) as per the requirements

Screenshot 2023-06-15 at 22 33 49

File: src/styles/app.module.css [hunk 2]
```diff
@@ -2,14 +2,38 @@
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  font-family: Montserrat, 'Times New Roman', Times, serif;
+}
+
+nav {
+  padding: 1.3rem;
```
Comment (Ghiftee) [inline src/styles/app.module.css]: - [ ] Kindly ensure that your app has the same spacing between elements as the given design and that the missing designs highlighted below are added.

Screenshot 2023-06-15 at 22 20 24

File: src/styles/BookList.module.css [hunk 3]
```diff
@@ -5,34 +5,108 @@
...
-  margin: 2rem 1rem;
+  padding: 0 4rem 4rem 4rem;
+  background-color: rgb(200, 200, 200, 0.1);
+}
+
+.form {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 1rem;
+}
```
Comment (Ghiftee) [inline src/styles/BookList.module.css:52]: - [ ] Kindly ensure that the elements spacing, the color and the size of the input fields and button in your form matches the given design.
- [ ] Also, please add the horizontal line highlighted in red as it is a part of the design

### Your design
Screenshot 2023-06-15 at 22 20 05

### Given design

Screenshot 2023-06-15 at 22 41 37

**Review by Ghiftee — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️♻️
Hello @forLoop94 👋🏽 

Good job working on the project so far! 
To highlight 👍🏽 
✔️ App runs smoothly
✔️ No linter errors
✔️ No inline styling

However, there are some issues that you need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review 👇🏽 _

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by divinecharlotte — APPROVED** [review]
# Hello @forLoop94      😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say  👍🏻 ✅ 
### To Highlight 💯


- Descriptive pr👍 
- Design 👍 
 




## Congratulations! 🎉 



Cheers, and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**